src/task/tenth/MyHashMap.java

```diff
@@ -0,0 +1,118 @@
+package task.tenth;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public class MyHashMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int size;
+    @SuppressWarnings({"rawtypes", "unchecked"})
+    private Node<K, V>[] entries = new Node[DEFAULT_INITIAL_CAPACITY];
```

Comment:
try to move initialization in the method put()

does we need @SuppressWarnings() here?